dot/rpc/subscription/listeners_integration_test.go

```diff
@@ -189,8 +189,8 @@ func TestExtrinsicSubmitListener_Listen(t *testing.T) {
 	wsconn.BlockAPI = BlockAPI
 
 	TxStateAPI := NewMockTransactionStateAPI(ctrl)
-	TxStateAPI.EXPECT().FreeStatusNotifierChannel(gomock.Any()).AnyTimes()
-	TxStateAPI.EXPECT().GetStatusNotifierChannel(gomock.Any()).Return(make(chan transaction.Status)).AnyTimes()
+	TxStateAPI.EXPECT().FreeStatusNotifierChannel(gomock.Any()).Times(1)
```

Comment:
The calls the has `Times(1)` you can leave it without the `Times(1)`, like:
```suggestion
	TxStateAPI.EXPECT().FreeStatusNotifierChannel(gomock.Any())
```